src/app/components/FilterButtons/index.tsx

```diff
@@ -21,7 +22,11 @@ export const FilterButtons = <T extends string>({ options, onSelect, value }: Fi
             clickable
             color="secondary"
             label={
-              <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+              <Box
+                component="span"
+                sx={{ display: 'flex', alignItems: 'baseline', justifyContent: 'center', gap: 2 }}
+              >
+                {selected && <CheckIcon fontSize={'small'} sx={{ transform: 'translateY(4px)' }} />}
```

Comment:
`alignItems: 'baseline' + 'translateY(4px)'` looks strange. how come not align center

Reply:
If I set "alignItems: center`, the vertical position of the buttons keep changing as I change the selection. As far as I could tell from a cursory look, this is because there is a component higher in the DOM  (some kind of card header I believe) that is also using baseline alignment, and adding the check-mark somehow messed it up. This issue doesn't occur when I use the baseline alignment.

Reply:
I really don't like patching bad css with bad css

Reply:
In any case, I expect that this component (along with the others) will be overhauled / replaced during the ui-library transition really soon, so as long as it's fully functional and looks good, I wouldn't worry about the long-term maintainability of the css code under the hood.

Reply:
I think we will remove this next sprint (uil replacement), so making such adjustments now is a waste of time